### PR TITLE
DOC: Update .zenodo

### PR DESCRIPTION
**Pull request by thewtex**
With the ITK/Utilities/Maintenance/UpdateZenodo.py script.

Add Lee N, Mihail I.

### CODE REVIEW

**Comment by thewtex** [comment]
@Leengit @issakomi congratulations -- you exceeded the 10 commit criteria for the Zenodo citation author list. Please check name, affiliation, ORCID.

File: .zenodo.json [hunk 1]
```diff
@@ -109,6 +114,11 @@
     {
       "name": "Hart, Gabe"
     },
+    {
```
Comment (dzenanz) [inline .zenodo.json:117]: This is now a duplicate of line 80.

File: .zenodo.json [hunk 2]
```diff
@@ -97,6 +97,11 @@
       "name": "Avants, Brian",
       "orcid": "0000-0002-4212-3362"
     },
+    {
```
Comment (dzenanz) [inline .zenodo.json:100]: This is duplication from line 12.

File: .zenodo.json (hunk 1 above)
Comment (dzenanz) [reply .zenodo.json:117]: I suspect the other additions are duplicates, too (except Lee and Mihail).

**Review by dzenanz — CHANGES_REQUESTED** [review]

File: .zenodo.json (hunk 1 above)
Comment (thewtex) [reply .zenodo.json:117]: Duplicate removed.

File: .zenodo.json (hunk 2 above)
Comment (thewtex) [reply .zenodo.json:100]: Duplicate removed.

Comment (Leengit) [reply .zenodo.json:100]:
```bash
fgrep '"name"' .zenodo.json | sort | uniq -c | egrep -v '^ *1 +'
```
gives
```plaintext
      2       "name": "Johnson, Hans",
      2       "name": "McCormick, Matthew",
      2       "name": "Straing, Marius"
      2       "name": "Tustison, Nick",
      2       "name": "Zuki\u0107, D\u017eenan",
```

File: .zenodo.json (hunk 1 above)
Comment (thewtex) [reply .zenodo.json]: I checked for duplicates on the others.

File: .zenodo.json (hunk 2 above)
Comment (Leengit) [reply .zenodo.json]: With caaea931fb82b9e20bddeaa9932a348896de9288 I am getting two duplicates:
```
      2       "name": "Straing, Marius"
      2       "name": "Tustison, Nick",
```

Comment (dzenanz) [reply .zenodo.json]: And Straing might be a misspelling of [Staring](https://scholar.google.com/citations?user=pKFkfq4AAAAJ&hl=en)

File: .zenodo.json [hunk 3]
```diff
@@ -350,6 +368,9 @@
     {
       "name": "C. Bigler, Don"
     },
+    {
+      "name": "Gerber, Samuel"
```
Comment (dzenanz) [inline .zenodo.json:372]: Affiliation: Kitware Inc

Comment (thewtex) [reply .zenodo.json:372]: Added.

**Comment by Leengit** [comment]
Duplicates in 3cede7ed9181bb50c402dc6c73960bb6b2040af2:
```
Straing, Marius
Tustison, Nick
```

**Comment by thewtex** [comment]
>Duplicates in 3cede7e:

These have been removed.

**Review by Leengit — APPROVED** [review]
LGTM

**Review by dzenanz — APPROVED** [review]
Thanks for the update.

**Review by issakomi — APPROVED** [review]

File: .zenodo.json [hunk 4]
```diff
@@ -193,6 +201,7 @@
       "orcid": "0000-0002-9688-8950"
     },
     {
+      "affiliation": "Kitware Inc",
```
Comment (Leengit) [inline .zenodo.json:204]: Do we want to change names from ALL CAPS family names for lines https://github.com/thewtex/ITK/blob/bbf4a978c032ef21f5b78f34273e3976e7efd99f/.zenodo.json#L210-L212 and lines https://github.com/thewtex/ITK/blob/bbf4a978c032ef21f5b78f34273e3976e7efd99f/.zenodo.json#L222-L225?

Comment (thewtex) [reply .zenodo.json:204]: No, these are the author's preference.

**Pull request merged**